types/chromecast-caf-receiver/index.d.ts

```diff
@@ -11,14 +11,18 @@
 /// <reference path="./cast.framework.system.d.ts" />
 /// <reference path="./cast.framework.ui.d.ts" />
 
+import * as framework from "./cast.framework";
 import { PlayerDataChangedEvent } from './cast.framework.ui';
-import { NetworkRequestInfo } from './cast.framework';
 import { Event } from './cast.framework.events';
 
 export as namespace cast;
-export type EventHandler = (event: Event) => void;
-export type PlayerDataChangedEventHandler = (
-    event: PlayerDataChangedEvent
-) => void;
-export type RequestHandler = (request: NetworkRequestInfo) => void;
-export type BinaryHandler = (data: Uint8Array) => Uint8Array;
+export { framework };
+
+declare global {
```

Comment:
This pattern is a bit unusual.  Is having consumers access the types through `cast` very inconvenient?

Reply:
nothing was 'global' before this PR. you had to `import { Sometype } from 'chromecast-caf-receiver' .. but as this is not an npm package, but a script reference that ends up on the `window` object, developers following the CAF documentation expected to be able to type `cast.framework.<whatever>` and get access to types.